packages/react-reconciler/src/ReactFiberCompleteWork.old.js

```diff
@@ -862,6 +862,14 @@ function completeWork(
     }
     case HostRoot: {
       const fiberRoot = (workInProgress.stateNode: FiberRoot);
+
+      if (enableTransitionTracing) {
+        const transitions = workInProgress.memoizedState.transitions;
+        if (transitions !== null) {
+          workInProgress.flags |= Visibility;
```

Comment:
I don't think this is the right flag. I would use Passive instead.

Reply:
Also please leave a comment explaining why this flag is being set. It's not obvious from context.

Reply:
Does passive work here? We would want to know when to run the transition tracing specific code somehow right (ie. when a suspense boundary toggles). If passive is set it doesn't necessarily mean that the suspense boundary has toggled.